src/js/components/FileInput/index.d.ts

```diff
@@ -22,10 +22,11 @@ export interface FileInputProps {
   };
   multiple?: boolean | { aggregateThreshold?: number; max?: number };
   name?: string;
+  onChange?: (event?: any, { files }?: { files: any }) => void;
```

Comment:
is there anything more specific than `any`

Reply:
Good comment, just updated this to make the types more specific